HoloPipelines/pipelineController.py

```diff
@@ -0,0 +1,101 @@
+import json
+import os
+import sys
+import argparse
+import subprocess
+import pathlib
+
+newCwd = str(pathlib.Path(str(os.path.dirname(os.path.realpath(__file__)))))
+
+parser = argparse.ArgumentParser(description="Selct pipeline to process")
+parser.add_argument(
+    "-c",
+    "--config",
+    default="./pipelineList.json",
+    type=str,
+    help="path to pipeline config file relative to pipelineController",
+)
+parser.add_argument(
+    "-l", "--ls", action="store_true", help="list all the available piplines"
```

Comment:
Typo `pipelines`
(Hint: using a full-fledged IDE like PyCharm makes life easier because it will point out such typos)

Reply:
Corrected. Yeah, I should look into using an actual IDE